panda/src/device/linuxInputDeviceManager.cxx

```diff
@@ -61,6 +61,9 @@ LinuxInputDeviceManager() {
 
     // We'll want to sort the devices by index, since the order may be
     // meaningful (eg. for the Xbox wireless receiver).
+    if (indices.empty()) { //Check if indices are empty
```

Comment:
This comment is redundant, because the code speaks for itself.  Comments should never be used to explain *what* the code does, only *why* the code does it.